<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlanClass</class>
 <widget class="QWidget" name="PlanClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>804</width>
    <height>765</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Plan</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="lbCalcTime">
         <property name="styleSheet">
          <string notr="true">QLabel {
    color: #333333; /* 文本颜色 */
    font-size: 14px; /* 字体大小 */
    font-weight: bold; /* 字体粗细 */
    font-family: Arial; /* 字体家族 */
}</string>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>距离</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="edBeginTime"/>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>到</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLineEdit" name="edEndTime"/>
           </item>
           <item>
            <widget class="QCheckBox" name="checkCurrTime">
             <property name="text">
              <string>当前时间</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QWebEngineView" name="widgetHtml" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>600</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>670</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header>qwebengineview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
